articles/quickstart/backend/php/01-authorization.md

```diff
@@ -1,251 +1,186 @@
 ```
 
-## Protect API Endpoints
 
-<%= include('../_includes/_api_endpoints') %>
+### Authenticating the user
 
-The `before` hook from the **router** package can be used to configure which routes are to be protected. For example, you may wish to protect all routes under a URL of `/api/private`.
+For this sample application, we're focusing on [authorization](https://auth0.com/intro-to-iam/authentication-vs-authorization/). There's numerous routes you could go for authenticating your users before they hit your backend API for authorization, such as using [Auth0's SPA.js library](https://github.com/auth0/auth0-spa-js). This approach is demonstrated in [this Quickstart app accompanying Github project](https://github.com/auth0-samples/auth0-php-api-samples/). Regardless of the approach you take, this sample application expects you to pass your Access Token to it through a request parameter or header to work.
 
-```php
-// index.php
+### Authorizing the request
+
+Our application will use what's called an "Authorization-bearer" scheme for informing our backend API of the user's access token from the frontend. When our frontend application sends requests to our new backend API, we'll want to include an `Authorization` header that includes the token. [You can read more about how this should look here.](https://auth0.com/docs/flows/call-your-api-using-the-authorization-code-flow#call-api)
+
+To simplify testing our sample application, we'll also accept the token from a `token` parameter during GET requests.
+
+```PHP
+// 👆 We're continuing from the steps above. Append this to your index.php file.
 
-// ...
-// Require composer autoloader
-require __DIR__ . '/vendor/autoload.php';
+$authorized = false;
+$token = $_GET['token'] ?? $_SERVER['Authorization'] ?? null;
+```
+
+Next, let's decode the token if one is present:
 
-$app = new \App\Main( 'https://${account.namespace}/', '${apiIdentifier}' );
+```PHP
+// 👆 We're continuing from the steps above. Append this to your index.php file.
 
-// Create Router instance
-$router = new \Bramus\Router\Router();
+// If a token is present, process it.
+if ($token !== null) {
+    // Trim whitespace from token string.
+    $token = trim($token);
 
-// Activate CORS
-function sendCorsHeaders() {
-  header("Access-Control-Allow-Origin: *");
-  header("Access-Control-Allow-Headers: Authorization");
-  header("Access-Control-Allow-Methods: GET,HEAD,PUT,PATCH,POST,DELETE");
+    // Remove the 'Bearer ' prefix, if present, in the event we're getting an Authorization header that's using it.
+    if (substr($token, 0, 7) === 'Bearer ') {
+        $token = substr($token, 7);
+    }
+
+    // The decode() method will parse our incoming token, validating it's structure. It will also verify its cryptographic signature, and validate the token's "claims", incoming the issuer, audience, expiration, and subject. It takes in a token and ensures it's valid for us to use with our app.
```

Comment:
I believe you meant "including" here
```suggestion
    // The decode() method will parse our incoming token, validating its structure. It will also verify its cryptographic signature, and validate the token's "claims", including the issuer, audience, expiration, and subject. It takes in a token and ensures it's valid for us to use with our app.
```